ci/pipelines/pact-broker.yml

```diff
@@ -35,6 +41,40 @@ resources:
       password: ((cf-password))
       cf_cli_version: 7
 
+  - name: pact-broker-src
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: PP-10768-adding-pact-broker-to-docker
+      paths:
+        - ci/docker/pact-broker/*
+
+  - name: pay-ci
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+
+  - name: pact-broker-ecr-registry-test
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/pact-broker
+      tag: "2.54.0.0"
```

Comment:
I think we should probably add a comment here to say that we're hardcoding this tag for now (and give a short summary of the reason why - we don't want to upgrade to a new version of pact broker while we're in the process of moving it). Once we're ready to upgrade we'll need to think about how we'll get the value for this `tag` field (this is a separate story).

Though just typing that, I've realised that Dependabot is going to try and upgrade the version in the `Dockerfile` as soon as we merge this PR... and it'll be tagged wrongly as 2.54.0.0! I'll make a note to snooze that Dependabot PR when it appears.